docs/advanced_examples/LogisticRegressionTraining.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Logistic Regression Training\n",
+    "# Logistic Regression Training over Encrypted Dataset\n",
```

Comment:
I have been told it's `on` and not `over` in this context.

Reply:
I would have kept in mind that both were ok, but ok I changed it to 'on'